client/layout/index.jsx

```diff
@@ -545,6 +549,7 @@ export default withCurrentRoute(
 			isGlobalSidebarCollapsed: shouldShowCollapsedGlobalSidebar && ! sidebarIsHidden,
 			isUnifiedSiteSidebarVisible: shouldShowUnifiedSiteSidebar && ! sidebarIsHidden,
 			isNewUser: isUserNewerThan( WEEK_IN_MILLISECONDS )( state ),
+			disableGlobalNotifications,
```

Comment:
Is there a good reason for this to be a prop? Looks like all it does is a check against an environment check, and I don't think there's a way for the value to change without a full app refresh.